Paymetheus/ViewModels/PassphraseDialogViewModel.cs

```diff
@@ -33,8 +33,9 @@ public PassphraseDialogViewModel(ShellViewModel shell, string header, string but
         {
             try
             {
-                await _execute(Passphrase);
-                HideDialog();
+                var success = await _execute(Passphrase);
```

Comment:
yes, that's possible too.  i just gave it a name since the function sig isn't documented anywhere (i should probably make it a named delegate type, and give it a doco comment) so it should be obvious if true or false means success or not.